packages/blob-storage-manager/src/storages/SwarmStorage.ts

```diff
@@ -63,16 +65,16 @@ export class SwarmStorage extends BlobStorage {
     return file.data.text();
   }
 
-  protected async _storeBlob(
-    chainId: number,
-    versionedHash: string,
-    data: string
-  ) {
+  protected async _removeBlob(reference: string): Promise<void> {
+    await this._swarmClient.bee.unpin(reference);
```

Comment:
This won't work as you mean. Content can't be removed from Swarm once you upload it. Unpinning only affects local storage of the bee node

Reply:
Noted. I'll leave it either way as it's better than doing anything I suppose